### ISSUE TITLE
if I have any sccache-dist build nodes, my build fails with `error: unknown type name '__u64'` in some /usr/include/linux header

### ISSUE THREAD

**Issue by dholbert** [mozilla/sccache#652]
Diagnosing with @froydnj and @chmanchester in slack, and filing this issue as a placeholder for when it's better understood.

tl;dr, as of the [sccache update this morning](https://hg.mozilla.org/mozilla-central/rev/2396b8073482ba138ca6129823fc6a756d417707), I cannot complete a mozilla-central build if I've got any sccache-dist builders on the network. (But my build completes just fine if I kill all the sccache-dist build nodes.)

I end up hitting an error like:
```

 2:31.77 In file included from Unified_cpp_linux0.cpp:38:
 2:31.77 In file included from /scratch/work/builds/mozilla-central/mozilla/toolkit/crashreporter/breakpad-client/linux/handler/exception_handler.cc:76:
 2:31.77 In file included from /scratch/work/builds/mozilla-central/obj/dist/system_wrappers/sys/prctl.h:3:
 2:31.77 In file included from /usr/include/x86_64-linux-gnu/sys/prctl.h:22:
 2:31.77 /usr/include/linux/prctl.h:135:2: error: unknown type name '__u64'
 2:31.77         __u64   start_code;             /* code section bounds */
 2:31.77         ^
```

(If I immediately try to rebuild, I get a similar same error for a few other directories, too [from parallel doomed-to-failure compilation jobs in different directories].  Some of these are for the same type name in the system header `/usr/include/linux/quota.h` instead of `prctl.h`.)

**Comment by emilio** [comment]
This is invalid (as in, it's an issue with glibc). This is basically the problem I described in #545, and it's fixed by https://sourceware.org/ml/libc-alpha/2019-11/msg00431.html.

**Comment by chmanchester** [comment]
As far as I can tell this is going to be a common issue, and the path to getting the fix isn't great. I think we need to default `rewrite_includes_only` to `false` until this gets into more distros. Nathan mentioned a configure check for the issue, which would be useful as well.
